src/mono/wasm/testassets/native-libs/undefined-symbol.c

```diff
@@ -0,0 +1,5 @@
+extern void sgfg();
```

Comment:
do we use this file ?

Reply:
Yes https://github.com/dotnet/runtime/blob/5c787a33b0dd664aa0742440f14b8a25bf1a535f/src/mono/wasm/Wasm.Build.Tests/NativeBuildTests.cs#L165